Source/Core/WinUpdater/Platform.cpp

```diff
@@ -255,12 +255,11 @@ std::optional<BuildInfos> InitBuildInfos(const std::vector<TodoList::UpdateOp>&
   if (op_it == to_update.cend())
     return {};
 
-  const auto op = *op_it;
-  std::string build_info_path =
-      temp_dir + DIR_SEP + HexEncode(op.new_hash.data(), op.new_hash.size());
+  const auto [_filename, old_hash, new_hash] = *op_it;
```

Comment:
Same here.

One could even argue that this needs `[[maybe_unused]]` to prevent unused variable warnings. Or use `std::tie` which can use `std::ignore`. Or wait for C++26 with [P2169R0](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2020/p2169r0.pdf) 🤷

Reply:
Neither of the two suggestions work.

Reply:
Just leave them without the underscore then. I don't really see a need to distinguish them in naming just because they're (intentionally) unused or not.